function-images/tests/chained-function-eventing/consumer/main.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 
-	. "eventing/eventschemas"
+	. "chained_function_eventing/eventschemas"
```

Comment:
i am not a big fan of such imports... they hurt readability as one cannot distinguish the module-internal code from the code resident in other modules. please avoid